vividus-plugin-rest-api/src/main/java/org/vividus/bdd/steps/api/JsonResponseValidationSteps.java

```diff
@@ -253,7 +253,7 @@ public void saveJsonElementToVariable(String json, String jsonPath, Set<Variable
      * </ul>
      * @param variableName A variable name
      */
-    @When("I set the number of elements found by the JSON path '$jsonPath' to the $scopes variable '$variableName'")
+    @When("I set number of elements found by JSON path `$jsonPath` to $scopes variable `$variableName`")
```

Comment:
Move step at line 162 to a composite like set and verify?

Reply:
I will be happy to do this once we have auto complete for composites in eclipse ;)

Reply:
we can't move such steps to composites, because we may choose variable name that conflicts with existing variables in the test projects

Reply:
we can assign variables names with hashes, like count_8277e0910d750195b448797616e091ad